websites/B/BigRigFM/dist/metadata.json

```diff
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Ninja",
+		"id": "150987570715688960"
+	},
+	"contributors": [
+		{
+			"name": "Pascal",
+			"id": "156560379089125376"
+		}
+	],
```

Comment:
Contributors must have contributed to at least 25% of the code base. I doubt this happened as the presence is new.

Reply:
The only reason I've added them is they contributed to the original code for the TruckersFM Presence (this is just a copy of that presence) I would link the commit though it was before the presences were moved to websites instead of Websites and can't see the commit.

I can remove it still if you wish though :)

Reply:
Please do

Reply:
Removed.